docs/how_to_guides/how_to_use_a_property_model.rst

```diff
@@ -0,0 +1,70 @@
+How to use a property model
+------------------------------------------------
+
+The example below shows how to use a property model and display outputs for a state block.
+
```

Comment:
May also be nice to have some information or language as to why someone may want or need to use a property model in the absence of a unit model.